sale_pricelist_discount_decimal_precision/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+Once installed, the module will add two new fields to pricelist items:
+percent_price and price_discount, both of which accept discount precision.
```

Comment:
Remove and change to: 

Go to Technical > Database Structure > Decimal Accuracy
Change the number of decimals for "Discount"
Pricelist will now accept discounts with the set number of decimals